lib/iris/plot.py

```diff
@@ -412,6 +412,16 @@ def _fixup_dates(coord, values):
         if coord.units.calendar == 'gregorian':
             r = [datetime.datetime(*date) for date in dates]
         else:
+            try:
+                import nc_time_axis
+            except ImportError:
```

Comment:
Of course, the problem with raising an error is that the failing examples will still fail, just with this error message instead...